FreshRSS/FreshRSS
# feat: create config to display website icon only / name only / icon and name / none on feeds

Closes #4838

Hello, I create this PR in draft, I'm open to review (codestyle, naming, other idea), it's my first real PR (with real code).

To be consistent with existing code for the `topline_thumbnail` (I copy it). I didn't use constants for `full`, `none`, `name` and `icon` values, but IMHO we should use constant for both config. If you are ok with this I can use constant for website and do the refactor for the thumbnail.

I want to create a enum for the values, but it's not available in PHP 7.

I don't know how translation works, I copy/past for all language (in no brain mode !). I see some po/pot files in project, maybe there is a translation tools ? if not I can add french translation in this PR directly.

I didn't run tests and quality tools, my dev env is not working, I will try again later. 

## Changes proposed in this pull request:

- Create a new display setting where you can select to display icon only or name only or icon and name or none for website on the feeds view

## How to test the feature manually:

1. Open the homepage `localhost:8080/i/` and check that website icon and label are visible by default.
2. Go to settings, check the new `Website` settings is here.
3. Change the website value and save.
4. Go back to homepage to verify that website display has changed.

## Screenshots
### The settings view :
<img width="1103" alt="Screenshot 2022-12-30 at 04 46 08" src="https://user-images.githubusercontent.com/5607440/210032468-08777d70-206b-4d72-a613-86f235a114ab.png">

### Feed in `full` view (default) :
<img width="1109" alt="Screenshot 2022-12-30 at 04 51 37" src="https://user-images.githubusercontent.com/5607440/210032721-e41512c0-666b-4677-b14f-804e156472aa.png">

### Feed in `none` view :
<img width="1103" alt="Screenshot 2022-12-30 at 04 47 46" src="https://user-images.githubusercontent.com/5607440/210032549-17ca3051-7ed9-415c-a4d7-f7b47e3da390.png">

### Feed in `icon` view :
<img width="1107" alt="Screenshot 2022-12-30 at 04 48 19" src="https://user-images.githubusercontent.com/5607440/210032574-d0be6426-9736-457a-b770-48e0ecf1ebbb.png">

### Feed in `name` view :
<img width="1110" alt="Screenshot 2022-12-30 at 04 51 03" src="https://user-images.githubusercontent.com/5607440/210032691-a2983d5b-f369-4f5f-a250-9f43f7dcda1f.png">

## Pull request checklist:


[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).
